depthai_nodes/ml/parsers/mediapipe_palm_detection.py

```diff
@@ -192,9 +192,12 @@ def run(self):
 
             bboxes = np.clip(bboxes, 0, 1)
             points = np.clip(points, 0, 1)
+            angles = np.round(angles, 0)
 
             detections_msg = create_detection_message(
-                bboxes=bboxes, scores=scores, angles=angles, keypoints=points
+                bboxes=bboxes, scores=scores, angles=angles
```

Comment:
is removing `keypoints=points` wanted here?

Reply:
Mistake, I was testing something and removed it.